pdsfile/pdsfile.py

```
@@ -1915,13 +1915,15 @@ def index_pdslabel(self):
-                                                  cls.LBL_EXT.upper())
-            try:
-              self._index_pdslabel = pdsparser.PdsLabel.from_file(label_abspath)
-            except:
-              self._index_pdslabel = 'failed'
+            for ext in cls.LBL_EXT:
+                label_abspath = self.abspath.replace (cls.IDX_EXT, ext)
+                label_abspath = label_abspath.replace(cls.IDX_EXT.upper(), ext.upper())
+                try:
+                    self._index_pdslabel = pdsparser.PdsLabel.from_file(label_abspath)
+                    break
+                except:
```

Flake8 (7.3.0)

[error] do not use bare 'except'

(E722)